.github/workflows/pr-assess-bundle-size.yml

```diff
@@ -50,6 +50,8 @@ jobs:
               with:
                   repo-token: '${{ secrets.GITHUB_TOKEN }}'
                   pattern: './{packages/js/!(*e2e*|*internal*|*test*|*plugin*|*create*),plugins/woocommerce-blocks}/{build,build-style}/**/*.{js,css}'
+                  install-script: 'pnpm install --filter="@woocommerce/plugin-woocommerce..." --frozen-lockfile --config.dedupe-peer-dependents=false'
+                  build-script: '--filter="@woocommerce/plugin-woocommerce" build'
```

Comment:
Not directly related to the PR, but `Create issues for flaky tests` job should not be triggered for this PR.

Reply:
TBD: add `BROWSERSLIST_IGNORE_OLD_DATA: true` to the job environment.